index.html

```diff
@@ -25,7 +26,9 @@
 
         <section class="intro">
             <h1>I'm Mucha <br> Glad to see <br> you!</h1>
-            <p>I'm a software developer! I can help you <br> build a product, feature or website Look <br> through some of my work and <br> experience! If you like what you see and <br> have a project you need coded, don't <br> hestiate to contact me.
+            <p>I'm a software developer! I can help you <br> build a product, feature or website Look <br> through some
+                of my work and <br> experience! If you like what you see and <br> have a project you need coded, don't
+                <br> hestiate to contact me.
             </p>
 
             <a href="#" class="con">LET'S CONNECT</a>
```

Comment:
- [OPTIONAL] As suggested by the previous reviewer. kindly ensure the content of the section is properly positioned inside the parent container  for better user experience. 

![optional](https://user-images.githubusercontent.com/26597432/219626589-aa269930-5329-42c3-98a8-dac6bb40dc10.png)